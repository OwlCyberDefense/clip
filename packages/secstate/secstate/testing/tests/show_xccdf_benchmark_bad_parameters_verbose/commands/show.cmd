/usr/sbin/secstate show PassComps -v
