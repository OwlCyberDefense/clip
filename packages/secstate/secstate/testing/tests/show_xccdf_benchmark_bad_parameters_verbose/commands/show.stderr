No such file: 'PassComps'
Error importing content: PassComps
